extra/1-currency-conversion.js

```diff
@@ -15,7 +17,9 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(amount) {
+  return parseFloat((amount * 0.99 * 5.7).toFixed(2));
```

Comment:
This is a clever solution and I like your use of parseFloat, well done! However, it is not that readable. Could you break it down into steps - perhaps working out the price after fee first. 

Also, could you be more specific with the labelling? i.e. instead of amount, GBP